### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by jmartinez95** [jmartinez95/MI-449-SS17-741-html-boilerplate-linting-and-code-style-uRxwab#1]
@stuartpearman Can you take a look at this? It's [hosted here](https://jmartinez95.github.io/MI-449-SS17-741-html-boilerplate-linting-and-code-style-uRxwab/) and meets the following criteria:

- [x] The files from your last project are copied and pasted into the directory for this project
- [x] Your code returns no errors (warnings are OK) when pasted into the [W3C Validator](https://validator.w3.org/#validate_by_input) (this will also be expected for all future projects)
- [x] Consistent spacing and indentation are used on every line (this will also be expected for all future projects)
- [x] `skeleton.css` is used to improve the design of the page

**Comment by stuartpearman** [comment]
Hi Jacob,

The indentation looks good, but this one's not quite there yet. Please make sure you have the criteria done before checking it complete, and read the lesson content that goes alongside the project. Here are a few things to adjust:

### Document head

At the top of your HTML document, you need to include `<head>` tags. In the head, you'll put a `title` within the head tags, and you'll declare a `character set` using a `<meta>` tag.

### Body tags

You'll a want opening `<body>` and closing `</body>` tags to go after the `head`. This is where you'll put any HTML elements that will show up on the page (paragraphs, headings, etc).

### Doctype Declaration

Basically, this tells the browser what type of document your file is.

All of the above information can be found in the first slide of the lesson. Start there if you are lost.

### Skeleton.css

Add skeleton.css to your project. You can find how to do this in the final lesson slide.

Add a comment to this thread when you're done and I'll be able to check your work again.

-Stuart

**Comment by jmartinez95** [comment]
Ah, only commited to gh-pages

**Comment by stuartpearman** [comment]
Ok, looks much better Jacob! The only thing that you're missing now is consistent spacing and indentation. Make sure that you are using some sort of consistent formatting. If you're stumped as to where the inconsistencies are, let me know and I can give you a hint :)

**Comment by jmartinez95** [comment]
@stuartpearman all set.

**Comment by stuartpearman** [comment]
Hey Jacob, this is looking better. One thing though, you'll generally want to use indentation to show nesting rather than to break up sections. So your `img` and `p` tags should be on the same level as your `h1`.

If you'd like to break your code into sections, there is a `<section></section>` tag that works great for this.

One more thing, in general, you don't want to put your `img` tags inside your paragraphs unless they are part of the content. So if you had an icon or something, that would be a time when you _would_ want to use an `img` within a paragraph.

**Comment by jmartinez95** [comment]
@stuartpearman fixed

**Comment by stuartpearman** [comment]
:shipit:
